lib/Backend/arm/EncoderMD.cpp

```diff
@@ -2250,7 +2250,7 @@ EncoderMD::BaseAndOffsetFromSym(IR::SymOpnd *symOpnd, RegNum *pBaseReg, int32 *p
             else
             {
                 AssertMsg(stackSym->IsAllocated(), "StackSym offset should be set");
-                Assert((uint)offset > func->m_argSlotsForFunctionsCalled * MachRegInt + func->GetMaxInlineeArgOutSize());
+                Assert(offset > func->HasTry() ? (int32)func->GetMaxInlineeArgOutSize() : func->m_argSlotsForFunctionsCalled * MachRegInt + func->GetMaxInlineeArgOutSize());
```

Comment:
Same here, the precedence of the ternary operator is low, so this doesn't have the intended meaning.